src/nupic/types/Sdr.hpp

```diff
@@ -318,6 +319,7 @@ class SparseDistributedRepresentation : public Serializable
      * the flattened SDR space.  This overwrites the SDR's current value.
      *
      * @param value A vector of flat indices to copy into the SDR.
+     *              Sparse data must be sorted and contain no duplicates.
```

Comment:
just double checking, I see a lot of `sort()`s and not `is_sorted`, so does the SDR,TM,SP implicitly sort the incoming data for you (text can be changed that output SDR sparse is always a sorted array), or does it just check (and text shoul add "and throws err otherwise")

Reply:
In C++ the SDR just checks and throws errors.

In Python it will sort for you.

Reply:
ok, so just a nit, may add @throws Sparse data must be sorted and contain no duplicates.